LiveController_deployment.md

```diff
@@ -0,0 +1,91 @@
+## Machine prerequisites:
+====================================================
+liveController & liveRecorder can be installed in single or separate machines.
+
+## Software prerequisites:
+====================================================
+ - RHEL/CentOS 7 or above or Ubuntu 16.0.4 or above
```

Comment:
We support both RHEL/CentOS 6 and 7. Both must work.
Ubuntu is not used for OP so it's not mandatory but:
A. there is not Ubuntu 16.0.4, there is 16.04
B. why do you think it won't work with say.. Ubuntu 14.04?